modules/apps/data-engine/data-engine-test/src/testFunctional/tests/wem/fields/GeolocationField.testcase

```diff
@@ -69,8 +95,23 @@ definition {
 	test DuplicateGeolocation {
 		property portal.acceptance = "false";
 
-		// TODO LPS-98071 DuplicateGeolocation pending implementation
+		var key_Label = "Localization";
+
+		var key_fieldLabel = "Copy of ${key_Label}";
+
+		DataEngine.addField(
+			fieldFieldLabel = "${key_Label}",
+			fieldName = "Geolocation");
+
+		DataEngine.editFieldRepeatable(fieldFieldLabel = "${key_Label}");
+
+		DataEngine.duplicateField(fieldFieldLabel = "${key_Label}");
```

Comment:
Method duplicateField (after method name in DataEngine.macro) is using fieldsetLabel  instead of fieldFieldLabel